core/server/models/user.js

```diff
@@ -10,7 +10,6 @@ var _              = require('lodash'),
     validation     = require('../data/validation'),
     events         = require('../events'),
     i18n           = require('../i18n'),
-    toString       = require('lodash.tostring'),
```

Comment:
This is deliberate because the version of lodash included in Ghost (3.10.1) does not have `_.toString` - it is added in a later version.